tests/simulator/refinement/test_2d_overlaps.py

```
@@ -0,0 +1,188 @@
+
+    def bx(x, y):
+        Lx = sim.simulation_domain()[0]
+        Ly = sim.simulation_domain()[1]
+        sigma = 1.0
+        dB = 0.1
+
+        x0 = x - 0.5 * Lx
+        y1 = y - 0.3 * Ly
+        y2 = y - 0.7 * Ly
+
+        dBx1 = -2 * dB * y1 * np.exp(-(x0**2 + y1**2) / (sigma) ** 2)
```

Ruff (0.11.9)

Local variable `dBx1` is assigned to but never used

Remove assignment to unused variable `dBx1`

(F841)